bundle/manifests/fence-agents-remediation.clusterserviceversion.yaml

```diff
@@ -11,6 +11,7 @@ metadata:
             "name": "worker-1"
           },
           "spec": {
+            "remediation_strategy": "OutOfServiceTaint",
```

Comment:
Please also commit the changes under [`config/samples`](https://github.com/medik8s/fence-agents-remediation/blob/main/config/samples/fence-agents-remediation_v1alpha1_fenceagentsremediation.yaml#L5)

Reply:
the field name is `remediationStrategy` not `remediation_strategy`

Reply:
how this file was created? by `make bundle`? 
CI should catch this kind of error if the bundle target hasn't been used

Reply:
PEBKAC, sorry. I forgot the link between the `config/sample` and what goes into `alm-examples`.